src/Confix.Tool/src/Confix.Tool/Variables/JsonVariableRewriter.cs

```diff
@@ -0,0 +1,66 @@
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Json.More;
+
+namespace ConfiX.Variables;
+
+public sealed class JsonVariableRewriter
```

Comment:
Dont we have already a rewriter?
I mean you essentially just have to overriide the "Rewrite" of the JsonDocumentRewriter